lib/disasters.js

```diff
@@ -145,7 +163,82 @@ function getDisaster(disasterId, callback) {
   });
 }
 
+async.auto({
+  disasters: ['operations', cacheDisasters],
+},
+function (err, results) {
+  if (err) {
+    console.log("hid_profiles cron run failed.");
+  }
+  else {
+    console.log("Finished hid_profiles cron run successfully.");
+  }
+  process.exit(err ? 1 : 0);
+});
+
 
+function newDisasterNotification(disaster) {
+  var requiredLocations = [];  
+  async.series([
+    function (cb) {
+      if(disaster.countries.length > 0){
+        async.forEachSeries(disaster.countries, function(country, callback2) {
+          if (!requiredLocations[iso3Countries[country]]) {
+                requiredLocations[iso3Countries[country]] = {};
+                requiredLocations[iso3Countries[country]].glide = [];
+                requiredLocations[iso3Countries[country]].contacts = [];
+                requiredLocations[iso3Countries[country]].disasterName = [];
+              }  
+            requiredLocations[iso3Countries[country]].glide.push(disaster.glide_id);
+            Contact.find({ $query: { type: 'local', location: iso3Countries[country]},  $orderby : {location: -1} }, function (err, contact){
+              contact.forEach(function(contact){
+                if(!contact){
```

Comment:
Checking if there are contacts returned should be done before the forEach, otherwise the forEach will trigger an error